go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cockroachdb/cockroach-go/v2
 go 1.13
 
 require (
-	github.com/gofrs/flock v0.8.1 // indirect
+	github.com/gofrs/flock v0.8.1
```

Comment:
Just from `go mod download; go mod tidy` on my machine, happy to revert if I got it wrong.